controllers/reconcilers/grafana/grafana_service_reconciler.go

```diff
@@ -17,13 +17,21 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+// .svc suffix needed for automatic openshift certificates: https://docs.openshift.com/container-platform/4.17/security/certificates/service-serving-certificate.html#add-service-certificate_service-serving-certificate
+const defaultClusterLocalDomain = ".svc"
```

Comment:
A couple of points:
1. I feel like it's a bit misleading. cluster domain is `cluster.local`, not `.svc` or `.svc.cluster.local`.

2. Later in code, you mentioned:
> By default, this is set to `.svc`, which should be fine for most deployments, but depending on network agents and frameworks, you may
> want to change this to the FQDN domain `.svc.cluster.local`.
> The default setting accommodates Openshift's default certificate signing behavior.

Which means the `.svc` part doesn't change, so we only need to offer customization for the trailing part (`cluster.local`).

Reply:
fair point, i have updated to always include .svc on the address and the domain part is just the optional last bit. this raises the question of if it's actually more appropriate to make this a boolean to make it use the FQDN or not.

is `useInternalFQDN` too broad?